.github/workflows/formatting-and-tests.yml

```diff
@@ -57,5 +57,9 @@ jobs:
                 dotnet-version: 6.0.x      
 
           - name: Run Tests
-            run: dotnet test
+            run: |
+              echo "Dotnet version: "
+              dotnet --version
```

Comment:
I have no idea how this managed to get the tests passing but I cannot get them to fail locally at all so I'm going to assume it's some GH weirdness and ignore!

Reply:
Sure! Can't hurt, right? RIGHT? 😬